setup.py

```diff
@@ -46,8 +47,8 @@
 # The version components are changed from ints to strings, but only VERSION
 # seems to matter outside of this module and it was already a str.
 FULLVERSION = versioneer.get_version()
-ISRELEASED = 'dev' not in FULLVERSION
-MAJOR, MINOR, MICRO = FULLVERSION.split('.')[:3]
+ISRELEASED = re.search(r'(dev|\+)', FULLVERSION) is None
```

Comment:
Help ourselves understand this in the future with a comment:
```suggestion
# Capture the version string: 
# 1.22.0.dev0+ ... -> ISRELEASED == True, VERSION == 1.22.0
# 1.22.0rc1+ ... -> ISRELEASED == False, VERSION == 1.22.0rc1
ISRELEASED = re.search(r'(dev|\+)', FULLVERSION) is None
```

Reply:
The first of those examples is incorrect :) Note that previously we reset the version string after a release so that the `rc1` would be missing.